src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -65,14 +68,23 @@ private static void setupDrivers(Application application) {
 
 		driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
 		DriverFeature.addDriver("Special line Simulator", driver);
+		
+		application.addTest("Custom Command", action -> CommandsFeature.getDriverCommandManager()
+				.getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver()));
+
```

Comment:
application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));

Reply:
I'm not sure what i was supposed to do.
These two lines do the same thing:

-application.addTest("Run command", new SelectRunCurrentCommandOptionListener(DriverFeature.getDriverManager()));

-application.addTest("Custom Command", action -> CommandsFeature.getDriverCommandManager()
.getCurrentCommand().execute(DriverFeature.getDriverManager().getCurrentDriver()));

but "Run Command" was already present in "User Test" tab, so i only removed our line.